encoding.h

```diff
@@ -166,11 +167,19 @@
 #define PTE_A     0x040 // Accessed
 #define PTE_D     0x080 // Dirty
 #define PTE_SOFT  0x300 // Reserved for Software
-
+#define PTE_PPN 0x3ffffffffffc00
```

Comment:
Why is the PPN constant?

Reply:
This is a mask to extract physical page number from pte entry,

Reply:
Oh, give it a name that indicates that it’s a mask